websites/J/JKAnime/presence.ts

```diff
@@ -0,0 +1,75 @@
+const presence = new Presence({
+    clientId: "731164832426098748"
+  }),
+  strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused",
+    browsing: "presence.activity.browsing"
+  });
+let video = {
+  duration: 0,
+  currentTime: 0,
+  paused: true
+};
+
+function getTimestamps(
+  videoTime: number,
+  videoDuration: number
+): Array<number> {
+  const startTime = Math.floor(Date.now());
+  const endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  return [Math.floor(startTime / 1000), (Math.floor(endTime))];

```

Comment:
Codacy found an issue: [Expected linebreaks to be 'LF' but found 'CRLF'.](https://app.codacy.com/gh/PreMiD/Presences/pullRequest?prid=6113512)